python.d/ceph.chart.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+# Description: ceph netdata python.d module
+# Author: Luis Eduardo (lets00)
+
+import rados
```

Comment:
wrap it in `try/except`
like
https://github.com/firehol/netdata/blob/master/python.d/beanstalk.chart.py#L5-L9

Reply:
Ok, this avoid ImportError Exception. I will fix this.